tools/wheel_resolver/__init__.py

```diff
@@ -0,0 +1,133 @@
+import click
+import typing
+import requests
+import os
+import urllib.request
+import logging
+import click_log
+import sys
+import tools.wheel_resolver.wheel as wheel
+import packaging.tags as tags
+import distlib.locators
+
+_LOGGER = logging.getLogger(__name__)
+
+click_log.basic_config(_LOGGER)
+
+def try_download(url):
```

Comment:
Strongly believe this function should go under `main` because right now it's in the middle of the `click` configuration

Reply:
Ive put it in its own output.py